cmd/tailscale/main.go

```diff
@@ -197,12 +197,16 @@ type SetLoginServerEvent struct {
 	URL string
 }
 
+// Web login event. If there is a URL, set the login server first.
```

Comment:
per https://go.dev/wiki/CodeReviewComments#comment-sentences this should be a complete sentence

And you never say what the URL is a URL of. If there's a URL of what, set the login server first?

```go
// WebAuthAndMaybeSetServerEvent is a web login event, after the
// user [does _________].
type WebAuthAndMaybeSetServerEvent struct {
    // URL optionally specifies a URL of _______.
    // If set, ___________.
    URL string
}
```